enhancements/audit-command.md

```diff
@@ -0,0 +1,722 @@
+- Be able to easily check and identify aspects of the operators bundles which will be published on it such as:
+    - check if they are using deprecated apis
+    - check if they are built with sdk and its versions
+    - the language used 
+    - check if they are multi-arch
+    - check if they are using some relevant feature provided by looking their annotations
+    - check if they are following the recommendations and conventions defined
+    - check all warnings and errors which will be raised by using the SDK features scorecard and bundle validate
+- Be able to extract a report with the audit results and in some formats such as json. 
+- Be able to run specific custom checks for each bundle in order to audit the catalog
+- Provide a documentation over how to use the audit command
+
+
+### Non-Goals
+
+- Add new checks or validators. This proposal has the goal to define the command and operation only. The checks should be addressed in their respective proposals.
+- Allow users select what specific validators will be used by the SDK commands. However, it can be addressed in the future.
+- Provide documentation that will describe how to add the custom scorecard tests in the bundle done with [Writing Custom Scorecard Tests](https://sdk.operatorframework.io/docs/advanced-topics/scorecard/custom-tests/) manually for Non-SDK projects or bundles which are not built with `make bundle` and how they are used with the audit command.
+ 
+**IMPORTANT**: Note that we are describing here an analytic tool/command that will start to be implemented in **alpha**. This feature uses what is in place and implemented already and cover needs which are raised in other proposals such as [Custom operator validation #43](https://github.com/operator-framework/enhancements/pull/43). However, it does not make any change or requires any new behaviour for the operator authors. The possibilities covered by the audit command can be easily documented after the first version of it this command be in place. 
+
+## Proposal
+
+Add a new command `audit` with the purpose to audit all desired operator bundle data of a specific index catalog. 
+
+By default, the command output a report where it is possible to have a view of the main characteristics and aspect of all operator bundles shipped in a specified OLM index catalog as check if they are passing in criteria and checks required to ensure their quality. However, accordingly with the purpose of its usage is also possible customize what should or not be done by this operation. 
+
+This command will perform checks by using the [SDK](https://github.com/operator-framework/operator-sdk) available sub commands; `bundle validate` and `scorecard bundle`.
+
+In this way, the binary will provide the alpha audit sub-command which would like: 
+
+```
+$ alpha audit --help
+
+Flags:
+  -h, --help                     help for audit
+  -i, --index-image              index image to get package from. 
```

Comment:
I think the index image will always be required correct? Seems it should be a required argument and not a flag. 

```
alpha audit <index-image> [flags]
```

Reply:
We can list the validators that can be used as well. So, I am not sure.